src/evict/evict_page.c

```diff
@@ -54,7 +54,8 @@ __wt_page_release_evict(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags)
 {
     WT_BTREE *btree;
     WT_DECL_RET;
-    uint32_t evict_flags, previous_state;
+    uint32_t evict_flags;
+    u_int previous_state;
```

Comment:
We're making the state a `uint8_t`. I prefer being explicit and using `uint8_t` everywhere rather than `u_int`.  I'm the kind of person who looks at `u_int` and thinks about "How big is that really? Do I need to know or worry about how big it is?" etc. With just `uint8_t` I don't have to think about it at all.

Reply:
Well, clang agrees with you -- fixed in f7d5969.